iron-overlay-manager.html

```diff
@@ -78,10 +78,58 @@
     },
 
     /**
+     * Brings the overlay at the specified index to the front.
+     * @param {number|Element} i
+     */
+    bringToFront: function(i) {
```

Comment:
I think it would be cleaner if we had explicit methods for different argument types. e.g., `bringToFront(HTMLElement)` and `brightToFrontByIndex(Number)` or something along those lines. WDYT?